### ISSUE TITLE
Proposal for adoption in metalsmith org

### ISSUE THREAD

**Issue by webketje** [ludofischer/metalsmith-gzip#15]
Hello @ludofischer , 

I'm Metalsmith's maintainer since Sept. 2021, have used this plugin to great effect previously and think it is essential enough to propose adopting this repo under the [metalsmith Github org](https://github.com/metalsmith). Especially since it could benefit from using Metalsmith's 2.4+ new features (`metalsmith.match` & `metalsmith.debug`)

If you agree, I will add you to the metalsmith org (for the create permission, see docs at https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#repository-transfers-and-organizations. Then you would need to transfer repo in the Settings > Danger Zone > Transfer repo. 

The plugin would be renamed to `@metalsmith/gzip` and would be aligned in setup (testing, dotfiles, etc)
When the first version of the new plugin is published, the "old" metalsmith-gzip on NPM would be marked as deprecated

If you agree, the LICENSE copyright notice would change to:
```
Copyright (c) 2023 webketje
Copyright (c) 2014-2023 Ludovico Fischer
```

... unless you're happy to keep maintaining this plugin (outside or as part of the metalsmith org)

Please let me know if any of this suits you!

**Comment by ludofischer** [comment]
Hello, sorry for the late reply. I had no idea this project was still used, and that Metalsmith was still maintained. In principle I have no objections to eventually move it to another organization, but could you first open PRs on this repo so I get an idea of what you are planning?

**Comment by webketje** [comment]
There you go. If it is migrated to a core plugin it would also undergo repo alignment (mocha instead of tape, enforcing of ESlint & Prettier style, Github Actions, code coverage) and I would make it a CJS/ESM dual module with Typescript support too

**Comment by ludofischer** [comment]
Hello, I've published metalsmith-gzip 2.0 with your changes. I bumped the official minimum Node.js version to 18.12 since the Node.js version bump was breaking anyway but all older Node.js versions are unsupported and the package isn't probably going to be updated that often. Let me know if everything works fine.

**Comment by webketje** [comment]
Closing since this plugin is still maintained
